### PR TITLE
Add tests for product types for 3.1

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I added tests for product types for 3.1

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5678 Delete product type and remove attribute](https://app.clickup.com/t/2549495/SALEOR-5678)

**Review by Cloud11PL — APPROVED** [review]

**Review by eCzerniak — CHANGES_REQUESTED** [review]

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Please add SALEOR_1508 case because it is missing.

File: cypress/integration/configuration/productTypes.js [hunk 1]
```diff
@@ -115,7 +129,147 @@ filterTests({ definedTags: ["all"] }, () => {
...
+            .click()
+            .waitForRequestAndCheckIfNoErrors("@ProductTypeDelete");
+          getProductType(productType.id).should("be.null");
+          getProductDetails(product.id)
+            .its("body.data.product")
+            .should("be.null");
+        });
+    });
+
+    it("should be able to remove variant attribute from product type. TC: SALEOR_1506", () => {
```
Comment (eCzerniak) [inline cypress/integration/configuration/productTypes.js:193]: This case or 1507 sometime is failing.
image

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Also don't you think that it is not too much of TCs in one file?

**Comment by eCzerniak** [comment]
> Also don't you think that it is not too much of TCs in one file?

@karola312 what about that comment? I do not see any answer.

**Comment by karola312** [comment]
You're right, I will split this file, and fix 1507

File: cypress/integration/configuration/productTypes/attributesInProduductTypes.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            .click()
+            .wait("@UnassignProductAttribute");
+          getProductType(productType.id);
+        })
+        .then(productType => {
+          expect(productType.assignedVariantAttributes).to.be.empty;
+        });
+    });
+
+    it("As an admin I should be able to select attribute as variant selection. TC: SALEOR_1508", () => {
```
Comment (eCzerniak) [inline cypress/integration/configuration/productTypes/attributesInProduductTypes.js:137]:
```suggestion
    it("should be able to select attribute as variant selection. TC: SALEOR_1508", () => {
```

**Review by eCzerniak — APPROVED** [review]
Please change this one name of TC and it is ready to merge.

**Review by eCzerniak — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**